### PR TITLE
Point installation instructions to Knative docs [apro #752]

### PR DESCRIPTION
**Pull request by concaf**
I've pointed the installation section of our documentation to Knative docs which is where we plan on keeping the updated instructions at all times.

Also, I've replaced the AES mentions with Ambassador mentions for now because we currently have instructions to install Knative with Ambassador and not with AES. We plan on updating this in Knative docs once AES is released.

### CODE REVIEW

**Comment by scoyle391** [comment]
https://github.com/datawire/ambassador-docs/pull/208

**Comment by concaf** [comment]
@scoyle391 +1 to your changes!

File: user-guide/knative.md [hunk 1]
```diff
@@ -1,101 +1,24 @@
...
 #### Prerequisites
 
 - Knative now requires Kubernetes v1.14, as well as a [compatible kubectl](https://knative.dev/docs/install/knative-with-ambassador/)
 - `kubectl` v1.10 is also required. This guide assumes that you’ve already created a Kubernetes cluster which you’re comfortable installing alpha software on.
-- The Ambassador Edge Stack should be installed in your cluster. Follow the [installation guide](../install) for instructions on installing the Ambassador Edge Stack.
 
 #### Installation
 
-1. Install Knative:
+Install the latest Knative Serving with Ambassador handling traffic to your serverless applications by following the instructions [here](https://knative.dev/docs/install/knative-with-ambassador/).
```
Comment (scoyle391) [inline user-guide/knative.md]: @concaf this is the only place I had a question on. Could we change it to say "Serving with Ambassador to handle traffic"? Not sure if that maintains the technical accuracy or not.

Comment (concaf) [reply user-guide/knative.md]: @scoyle391 sure, we can change it to - `Install the latest Knative Serving with Ambassador to handle traffic to your serverless applications by following the instructions`

Sorry, but I couldn't find any comment from you earlier in the PR :( maybe due to new changes. Nvm.

**Pull request merged**